css/style.css

```diff
@@ -100,6 +97,252 @@ ul li a {
   }
 
   .small {
-    display: block;
+    display: inline-block;
+    padding-right: 30px;
+  }
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.fa-angle-down {
+  font-size: 25px;
+  padding: 5px 5px;
+  border: 1px solid rgba(0, 0, 0, 0.63);
+  border-radius: 50% 50%;
+}
+
+.work-title {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+}
+
+.work-title::after {
+  content: "";
+  display: block;
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  margin: 15px 45%;
+}
+
+/* cards layout (grid css layout) */
+@media only screen and (max-width: 600px) {
+  .card-layout {
+    display: grid;
+    gap: 1.2em;
+    grid-template-columns: 1fr;
+    justify-content: center;
+  }
+}
+
+@media only screen and (min-width: 600px) {
+  .card-layout {
+    display: grid;
+    gap: 1.2em;
+    grid-template-columns: 1fr 1fr 1fr;
+    justify-content: center;
+  }
+
+  body {
+    background-image: none;
+  }
+}
+
+/* cards (used flexbox to align item) */
+.card {
+  background-color: #ebf0ee;
+  height: 450px;
+  width: 310px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+/* card body */
+.card-body {
+  background-color: #fff;
+  bottom: 0;
+  height: 222px;
+  width: 310px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  border-radius: 8px;
+  padding: 0;
+  margin: 0;
+  gap: 6px;
+}
+
+.card-title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Well done on the project 🚀 but to make it match the Figma you should consider adding some margin-top to the title `(Muti-Post Stories Gain+Glory)`. See below 👇🏽 

## Your design
![Screenshot 2023-01-13 at 21 35 00](https://user-images.githubusercontent.com/45739963/212447010-8d4756a0-a433-4fdf-82b7-164a92260849.png)

## Figma Design
![Screenshot 2023-01-13 at 21 36 31](https://user-images.githubusercontent.com/45739963/212447064-8a72a603-458f-49e5-8e7b-5999ac75a825.png)